osu.Game.Rulesets.Mania.Tests/Skinning/TestSceneDrawableJudgement.cs

```diff
@@ -16,14 +17,17 @@ public class TestSceneDrawableJudgement : ManiaSkinnableTestScene
     {
         public TestSceneDrawableJudgement()
         {
+            var HitWindows = new ManiaHitWindows();
+
             foreach (HitResult result in Enum.GetValues(typeof(HitResult)).OfType<HitResult>().Skip(1))
```

Comment:
```suggestion
            foreach (HitResult result in new ManiaHitWindows(). GetAllAvailableWindows())
```

may work better (haven't tested, but looks to be what you want).

Reply:
Tested. Would end up causing the miss animation to be excluded

Reply:
Tested. Would end up causing the miss animation to be excluded